docs/agents/labjack.rst

```diff
@@ -32,10 +32,37 @@ available arguments::
          'arguments':[
            ['--ip-address', '10.10.10.150'],
            ['--num-channels', '13'],
+           ['--function-file', 'labjack_functions.yml'],
            ['--mode', 'acq'],
            ]},
 
-You should assign your LabJack a static IP, you'll need to know that here.
+You should assign your LabJack a static IP, you'll need to know that here. 
+The 'functions-file' argument specifies the labjack configuration file, 
+which is located in your OCS configuration directory. This allows analog 
+voltage inputs on the labjack to be converted to different units. Here is 
+an example labjack configuration file:
```

Comment:
Add another `:` to the end of `configuration file:` for proper formatting of the example configuration file.

Reply:
done